beam/www/beam/stores/scan.ts

```diff
@@ -19,7 +19,7 @@ export const useScanStore = defineStore('scan', () => {
 
 	const documentId = computed(() => {
 		const currentRoute = store.router.currentRoute.value
-		return currentRoute.params.id || currentRoute.query.id || ''
+		return currentRoute.params.id || currentRoute.query.id || store.router.currentRoute.value?.name || ''
```

Comment:
Would the following also work:

```suggestion
		return currentRoute.params.id || currentRoute.query.id || currentRoute.name || ''
```